ui/app/styles/pki/pki-beta.scss

```diff
@@ -0,0 +1,12 @@
+.pki-beta-modal-list {
+  list-style: disc;
+  padding: inherit;
+}
+.pki-beta-button {
+  border: none;
```

Comment:
Also needed these one-off styles for the button! The `New PKI UI Available` tag looks and acts like a link, but since there is an `action` involved it's a button!

Reply:
We might be able to use `.text-button` here and then add `is-underline` as a class to the component? But might be worth checking with Ivana if she wants it always underlined - I remember an accessibility discussion a while ago that mentioned we do want that, but I may be misremembering! 

otherwise we could add the following to text-button - this addition would be beneficial where the class is used already, 
```js
 &:hover {
    text-decoration: underline;
  }
```

just suggestions! if you think a separate class file for this button is necessary, I trust your judgement

Reply:
Love these ideas! I wasn't a big fan of making this new stylesheet! I didn't know that we had a `.bullet` class and a `text-button` class :) I could not find it!